sdk/src/genesis_config.rs

```diff
@@ -22,14 +22,14 @@ use {
     bincode::{deserialize, serialize},
     chrono::{TimeZone, Utc},
     memmap2::Mmap,
+    solana_patches::time::{SystemTime, UNIX_EPOCH},
```

Comment:
These are only used to create defaults for `GensisConfig`, so it should be fine to have dummy values.